doc/api/fs.markdown

```diff
@@ -397,7 +397,18 @@ If `options` is a string, then it specifies the encoding.
     Stability: 0 - Deprecated: Use [`fs.stat()`][] or [`fs.access()`][] instead.
```

Comment:
If we are going ahead, we should move the deprecation notice down to the backward compatibility note.